### PR TITLE
Fix 'generate-examples' command failed to generate correct format for ‘uri’ format parameter

### PR DESCRIPTION
**Pull request by Ligengxin96**

### CODE REVIEW

File: lib/generator/mocker.ts [hunk 1]
```diff
@@ -48,7 +48,14 @@ export default class Mocker {
       );
     }
     const length = this.getRandomInt(minLength, maxLength);
-    return "a".repeat(length);
+    let mockedValue = "a".repeat(length);
+
+    if (paramSpec.format === "uri") {
+      const prefix = "https://a";
```
Comment (raych1) [inline lib/generator/mocker.ts]: Can you add hostname like https://host.com/a? Also pls guarantee the generated value passes model validation.

**Review by raych1 — COMMENTED** [review]
Can you check if the spec validated in exampleGeneratorTests cover url scenario? Please add it if it's not covered.

File: test/__snapshots__/exampleGeneratorTests.ts.snap [hunk 2]
```diff
@@ -125,6 +125,85 @@ Array [
...
   ],
+  Object {
+    "code": "INVALID_FORMAT",
+    "exampleJsonPath": "$parameters.parameters",
+    "examplePosition": Object {
+      "column": 19,
+      "line": 8,
+    },
+    "exampleUrl": "test/exampleGenerator/specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/examples/ManagedDatabaseCompleteExternalRestore.json",
+    "message": "Object didn't pass validation for format uri: testdb1_log4",
```
Comment (raych1) [inline test/__snapshots__/exampleGeneratorTests.ts.snap]: Can you share why test still doesn't pass URL format validation?

Comment (Ligengxin96) [reply test/__snapshots__/exampleGeneratorTests.ts.snap]: Oh, I forgot update the example file.

**Review by raych1 — APPROVED** [review]

**Pull request merged**